__tests__/app.test.js

```diff
@@ -175,3 +175,62 @@ describe("Post comment", () => {
       });
   });
 });
+
+describe.only("Patch an article", () => {
+  test("200: Responds with the updated article", () => {
+    const newUpdate = { inc_votes: 50 };
+    return request(app)
+      .patch("/api/articles/1/")
+      .send(newUpdate)
+      .expect(200)
+      .then((response) => {
+        const { article } = response.body;
+        expect(article.votes).toBe(150);
+      });
+  });
+
+  test("200: Returns the same article unaltered if inc_votes = 0", () => {
+    const newUpdate = { inc_votes: 0 };
+    return request(app)
+      .patch("/api/articles/1/")
+      .send(newUpdate)
+      .expect(200)
+      .then((response) => {
+        const { article } = response.body;
+        expect(article.votes).toBe(100);
+      });
+  });
+
+  test.only("200: Works for negative votes", () => {
```

Comment:
Should remove .only's before pushing, we want to make sure all of your tests are passing!